pkg/client/listers/crd/v1alpha1/packetsampling.go

```diff
@@ -0,0 +1,66 @@
+// Copyright 2023 Antrea Authors
```

Comment:
```suggestion
// Copyright 2024 Antrea Authors
```
You need to regenerate these files to get the right year.

Reply:
the script seems still generate files with `2023` info. Do we have any official way to update this?  Manually  update these files seems not a good way.